src/dotnet/Chat.Service/ChatsBackend.cs

```diff
@@ -123,7 +123,6 @@ public virtual async Task<ApiArray<ChatId>> GetPublicChatIdsFor(PlaceId placeId,
         return sChatIds.Select(x => new ChatId(x)).Where(x => !x.IsPlaceRootChat).ToApiArray();
     }
 
-    // Non-compute methods
```

Comment:
Now it's a compute method, so let's add a comment

Reply:
Done